server/statistics/hot_peer_cache.go

```diff
@@ -454,3 +412,93 @@ func (f *hotPeerCache) updateHotPeerStat(newItem, oldItem *HotPeerStat, deltaLoa
 	}
 	return newItem
 }
+
+// TODO: remove it in future
+func (f *hotPeerCache) collectRegionMetrics(loads []float64, interval uint64) {
+	regionHeartbeatIntervalHist.Observe(float64(interval))
+	if interval == 0 {
+		return
+	}
+	// TODO: use unified metrics. (keep backward compatibility at the same time)
+	for _, k := range f.kind.RegionStats() {
+		switch k {
+		case RegionReadBytes:
+			readByteHist.Observe(loads[int(k)])
+		case RegionReadKeys:
+			readKeyHist.Observe(loads[int(k)])
+		case RegionWriteBytes:
+			writeByteHist.Observe(loads[int(k)])
+		case RegionWriteKeys:
+			writeKeyHist.Observe(loads[int(k)])
+		}
+	}
+}
+
+// gets the storeIDs, including old region and new region
+func (f *hotPeerCache) getAllStoreIDs(region *core.RegionInfo) []uint64 {
```

Comment:
What is the difference between it and the previous one?

Reply:
no difference